test-core-circle-ci.ini

```diff
@@ -0,0 +1,140 @@
+#
```

Comment:
Rather than duplicating the whole file, if for some reason you need to override a config setting from `test-core.ini` you can just create this new file:

```
[server:main]
use = egg:Paste#http
host = 0.0.0.0
port = 5000

[app:main]
use = config:test-core.ini

sqlalchemy.url = postgresql://ckan_default:pass@ckan-postgres/ckan_test

ckan.datastore.write_url = postgresql://datastore_write:pass@ckan-postgres/datastore_test
ckan.datastore.read_url = postgresql://datastore_read:pass@ckan-postgres/datastore_test
```